specs/mpx_auth_SUITE.erl

```diff
@@ -0,0 +1,157 @@
+-module(mpx_auth_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("deliverly/include/deliverly.hrl").
+-include_lib("deliverly/include/log.hrl").
+-include_lib("deliverly/include/priv.hrl").
+
+-import(test_utils, [
+  gun_open/1,
+  request_token/2,
+  request_token/3,
+  message/0
+]).
+
+-compile(export_all).
+
+init_per_suite(Config) ->
+  ulitos_app:set_var(?APP, redis_database, 7),
+  User = <<"admin">>,
+  Pass = list_to_binary(ulitos:random_string(8)),
+  ulitos_app:set_var(?APP, api_user, User),
+  ulitos_app:set_var(?APP, api_pass, Pass),
+  ulitos_app:set_var(?APP, default, #{}),
+  ulitos_app:set_var(?APP, mpx, #{}),
+  lager:start(),
+  deliverly:start(),
+  ulitos_app:ensure_started(gun),
+  [{user, User}, {pass, Pass}, {port, 8081} | Config].
+
+end_per_suite(_) ->
+  redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),
```

Comment:
Line 31 is too long:   redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),.